WORKSPACE

```diff
@@ -2,8 +2,8 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 http_archive(
     name = "io_bazel_rules_go",
-    urls = ["https://github.com/bazelbuild/rules_go/releases/download/0.13.0/rules_go-0.13.0.tar.gz"],
```

Comment:
Why was this changed ?

Reply:
+1, I think this was to fix the mac issue we ran into a month back

Reply:
@terenc3t about the reasons for this change see https://github.com/prysmaticlabs/prysm/pull/321#issuecomment-407689242, if `0.13.0` is required i'lll remove the concurrent write unit test and the disable the race detector again, it can be added once `0.13.0` (or a higher version) supports it.

Reply:
Please add a comment stating that rules_go should not be updated until that specific bug has been resolved.

The Mac issue that others were referring to had to do with bazel itself, not these rules.

Reply:
Ok, added comment with a link to the bug.